src/cc_bstring.c

```diff
@@ -142,3 +142,35 @@ bstring_atou64(uint64_t *u64, struct bstring *str)
 
     return CC_OK;
 }
+
+struct bstring *
+bstring_alloc(uint32_t size)
+{
+    struct bstring *bs = cc_alloc(sizeof(struct bstring));
```

Comment:
We need to check if the cc_alloc succeeded here. Also echoing what I said earlier re: sizeof().